### PR TITLE
travel perk model and service creation

### PR DESCRIPTION
**Pull request by DhaaraniCIT**

### CODE REVIEW

File: src/app/core/models/travelperk/travelperk.model.ts [hunk 1]
```diff
@@ -32,3 +33,37 @@ export type WorkatoConnectionStatus = {
         error?: string,
     }
 }
+
+export type TravelperkAdvancedSetting = {
```
Comment (ashwin1111) [inline src/app/core/models/travelperk/travelperk.model.ts:37]: move to separate files, both models

File: src/app/core/models/travelperk/travelperk.model.ts [hunk 2]
```diff
@@ -32,3 +33,37 @@ export type WorkatoConnectionStatus = {
...
+	import_to_fyle: boolean
+}
+
+export interface TravelperkProfileMappingGet extends TravelperkProfileMapping {
+    id: number,
+    created_at: Date,
+    update_at: Date,
+    workspace: number
+}
+
```
Comment (ashwin1111) [inline src/app/core/models/travelperk/travelperk.model.ts:68]: one more model for paginated response

File: src/app/core/services/travelperk/travelperk.service.ts [hunk 3]
```diff
@@ -74,4 +76,20 @@ export class TravelperkService {
   disconnect(): Observable<{}> {
     return this.apiService.post(`/orgs/${this.orgId}/travelperk/disconnect/`, {});
   }
+
+  getTravelperkPaymentProfileMapping(): Observable<TravelperkProfileMappingGet[]> {
```
Comment (ashwin1111) [inline src/app/core/services/travelperk/travelperk.service.ts:80]: type would change

**Comment by DhaaraniCIT** [comment]
@coderabbitai review

File: src/app/core/models/travelperk/travelperk-configuration/travelperk-payment-profile-settings.model.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+export interface TravelperkPaymentProfileSettingGet extends TravelperkPaymentProfileSetting {
+    id: number,
+    created_at: Date,
+    update_at: Date,
+    workspace: number
+}
+
+export interface TravelperkPaymentProfileSettingPost extends TravelperkPaymentProfileSetting { }
+
+export interface TravelperkPaymentProfileSettingGetPaginator extends Paginator {
```
Comment (ashwin1111) [inline src/app/core/models/travelperk/travelperk-configuration/travelperk-payment-profile-settings.model.ts:21]: TravelperkPaymentProfileSettingResponse

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**